docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -13,32 +13,63 @@ Teleport FIPS mode works and how it can help your company to become FedRAMP auth
 Teleport includes FedRAMP and FIPS 140-2 features to support companies that sell into
 government agencies.
 
-| Control | Teleport Features |
+### Access Controls
```

Comment:
```suggestion
### Access controls
```
Headings in the docs use sentence case: https://github.com/gravitational/docs/blob/main/docs-contributors/style-guide.md#headings